conf.py

```diff
@@ -393,7 +393,7 @@
 # FORCE_ISO8601 = False
 
 # Date format used to display post dates. (translatable)
-# Used by babel.dates, CLDR style: http://cldr.unicode.org/translation/date-time
+# Used by babel.dates, CLDR style: https://cldr.unicode.org/translation/date-time
```

Comment:
http: works "but page not found"
https: doesn't work

maybe we can use http://cldr.unicode.org/translation/date-time-1/date-time-patterns ?